docs/manual/developer/06_contributing_with_content.md

```diff
@@ -172,13 +172,16 @@ A rule itself contains these attributes:
     under `shared/checks/oval/` and referenced in the dictionary file. It is
     possible to specify multiple platforms in the list. In that case, they are
     implicitly connected with "OR" operator.
-    
+
+    Group(s) platforms are inherited by rules.
+    That's it, platforms of the group hierarcy would be implicitly joined with rule's platforms using "AND" operator.
```

Comment:
hierarchy

Reply:
Please reword this, I would use "This means that" instead "That's it,".

Reply:
Done